### PR TITLE
Editing Fariba's bio and adding the docker file 

### PR DESCRIPTION
**Pull request by frb-yousefi**
I changed my bio, added docker section to the README and added docker folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,7 +219,7 @@ The following project data (and metadata) are to be populated in `projects.csv`:
 | tech_methods | Technology and methods used in the project including programming languages. Each item in this list should be separated by a comma and a space. | Yes |
 | rses | Comma seperated list of RSEs involved (`firstname` `lastname`). | Yes |
 | start | Project start date `dd/mm/yyyy` | Yes |
-| end | Project end date `dd/mm/yyyy`; leave as the empty string if ongoing | No |
```
Comment (Robadob) [inline README.md]: Have you changed this line by accident?

The empty string/ongoing thing is used for "Teaching Support for GPU Module" on the [projects page](https://rse.shef.ac.uk/service/projects/).

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:20.04
```
Comment (willfurnass) [inline docker/Dockerfile:1]: Is this similar to what is proposed in PR #370?

**Comment by willfurnass** [comment]
Is this similar to what @tartarini proposed in #370?

File: README.md (hunk 1 above)
Comment (frb-yousefi) [reply README.md]: Probably, will double-check it! Thanks.

**Comment by frb-yousefi** [comment]
> Is this similar to what @tartarini proposed in #370?

I build the image from scratch and that one is using the predefined Jekyl image.

File: README.md (hunk 1 above)
Comment (frb-yousefi) [reply README.md]: Fixed it, the version that I had was like that, and I had forgotten to pull it before adding changes.

**Comment by davidwyld** [comment]
@frb-yousefi I would recommend putting this blog post on a separate branch and making a distinct PR for it.

**Comment by willfurnass** [comment]
@frb-yousefi Do you think this has been superseded by #432 and #433 and can be closed without merging?

**Review by benkaday3218 — COMMENTED** [review]
Merci

**Pull request merged**